sections/stats/Trades.tsx

```diff
@@ -0,0 +1,141 @@
+import { useEffect, useRef } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { useGetFuturesTradeStats } from 'queries/futures/useGetFuturesTradeStats';
+import colors from 'styles/theme/colors/common';
+import fonts from 'styles/theme/fonts';
+
+import { initBarChart } from './initBarChart';
+import type { EChartsOption } from './initBarChart';
+import { TradesWrapper, Wrapper } from './stats.styles';
+import { TimeRangeSwitcher } from './TimeRangeSwitcher';
+
+let chartInstance: any;
+
+export const Trades = () => {
```

Comment:
lines and bars are backwards. The bars should be the daily value, the line is the cumulative value.